site-src/v1alpha2/guides/tcp.md

```diff
@@ -43,6 +43,24 @@ In this way each `TCPRoute` "attaches" itself to a different port on the
 `Gateway` so that the service `my-foo-service` is taking traffic for port `8080`
 from outside the cluster and `my-bar-service` takes the port `8090` traffic.
 
+## Alternatives: Routing Using Traffic Matching
+
+**EXPERIMENTAL**
```

Comment:
Maybe use an [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#inline-blocks) for this? I'm not sure if this should be "info" or "warning". Something like this might work:

```
??? info "Experimental Channel"

    This functionality is currently only included in the "Experimental" channel
    of Gateway API. For more information on release channels, refer to the
    [related documentation](https://gateway-api.sigs.k8s.io/concepts/versioning/#adding-experimental-fields).
```

Reply:
added in bb3672a1517853dbbfec47b7c02a5b784d376bc1 :+1:

Reply:
@shaneutt apparently my snippet above must have relied on a newer version of the plugin or something. Replacing `???` with `!!!` seems to work though.

Reply:
Ok, now changed in 5f448f55bb08f3bb857fed145c336e8716969b45 :+1: